modules/ROOT/pages/configure-client-management-entra-id.adoc

```diff
@@ -51,17 +53,18 @@ Use the following properties when you are using the Authorization Code grant typ
 
 Anypoint Platform does not support custom scopes. Use the default scope for Microsoft Graph.
 
+[[grant-types]]
 === Grant Types
 
-For a list of supported OAuth 2.0 authorization code grant types, see the https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow[Microsoft identity platform documentation].
+For a list of supported OAuth 2.0 authorization code grant types, see the https://learn.microsoft.com/en-us/entra/identity-platform/v2-oauth2-auth-code-flow[Microsoft identity platform documentation].
 
-When https://docs.microsoft.com/en-us/graph/api/application-post-applications?view=graph-rest-1.0&tabs=http[creating a new application with Microsoft], you can't specify the supported OAuth 2.0 authorization code grant types. Because of this limitation on the Microsoft Microsoft Entra ID side, Anypoint Platform does not support selecting grant types.
+When https://docs.microsoft.com/en-us/graph/api/application-post-applications?view=graph-rest-1.0&tabs=http[creating a new application with Microsoft], you can't specify the supported OAuth 2.0 authorization code grant types. Because of this limitation on the Microsoft Entra ID side, Anypoint Platform does not support selecting grant types.
```

Comment:
```suggestion
When https://docs.microsoft.com/en-us/graph/api/application-post-applications?view=graph-rest-1.0&tabs=http[creating a new application with Microsoft], you can't specify the supported OAuth 2.0 authorization code grant types. Because of this limitation on the Microsoft Entra ID side, Anypoint Platform doesn't support selecting grant types.
```